buildbot/osuosl/master/config/builders.py

```diff
@@ -2522,7 +2522,7 @@
     'workernames' : ["minipc-ryzen-win"],
     'builddir': "flang-x86_64-windows",
     'factory' : UnifiedTreeBuilder.getCmakeWithNinjaBuildFactory(
-                    depends_on_projects=['llvm','mlir','clang','flang'],
+                    depends_on_projects=['llvm','mlir','clang','flang','flang-rt'],
```

Comment:
Does this change together with the addition to the frozenset below already enable the build of this runtime or will this require another patch to enable it?

Reply:
Thank you.